tests/test_widget.py

```diff
@@ -239,18 +268,3 @@ def set_directory_and_close(dialog):
         assert msgbox.text() == (
             "The data folder must contain three subfolders, one for each view, and each subfolder must contain the same number of images."
         )
-
-
-def test_show_hide_buttons(cpt_widget: ParticleTracksWidget):
-    """Test the show/hide buttons"""
-    assert cpt_widget.rad.isEnabled() is False
-    assert cpt_widget.lgth.isEnabled() is False
-    assert cpt_widget.ang.isEnabled() is False
-    cpt_widget.cb.setCurrentIndex(1)
-    assert cpt_widget.rad.isEnabled() is True
-    assert cpt_widget.lgth.isEnabled() is True
-    assert cpt_widget.ang.isEnabled() is False
-    cpt_widget.cb.setCurrentIndex(4)
-    assert cpt_widget.rad.isEnabled() is False
-    assert cpt_widget.lgth.isEnabled() is True
-    assert cpt_widget.ang.isEnabled() is True
```

Comment:
Why?

Reply:
This is my fault, got duplicated somehow when I was cherry picking, the final test is further up.